versions.yaml

```diff
@@ -40,35 +52,7 @@ versions:
       mongodb:
         gpg: *gpg36
         major: '3.6'
-        version: '3.6.8'
-  - dir: '3/debian8/3.4'
-    repo: mongodb3
-    tags:
-      - '3.4.17-debian8'
-      - '3.4-debian8'
-      - '3.4.17'
-      - '3.4'
-    from: *from8
-    packages:
-      gosu: *gosu
-      mongodb:
-        gpg: *gpg34
-        major: '3.4'
-        version: '3.4.17'
-  - dir: '3/debian8/3.2'
-    repo: mongodb3
-    tags:
-      - '3.2.21-debian8'
```

Comment:
Please mark tags `3.2` and `3.4` as **deprecated** first.

Reply:
Marked already